demo/angularjs/components/search/doc.md

```diff
@@ -0,0 +1,17 @@
+# Search
+
+**Search allow users to specify a word or a phrase to find particular relevant pieces of content.**
```

Comment:
`particular` and `relevant` are redundant, `pieces of content` is the proper way but `relevant content` would be acceptable too.